src/memory-analyzer/analyze_symbol.cpp

```diff
@@ -433,11 +440,22 @@ exprt gdb_value_extractort::get_non_char_pointer_value(
 }
 
 bool gdb_value_extractort::points_to_member(
-  const pointer_valuet &pointer_value) const
+  pointer_valuet &pointer_value,
+  const typet &expected_type)
 {
   if(pointer_value.pointee.find("+") != std::string::npos)
     return true;
 
+  if(pointer_value.pointee.empty())
```

Comment:
Well `pointer_value` is the result of the gdb query `get_memory`. When the `pointee` is empty it means that gdb did not know what _structure_ the pointer points to. This can happen if the pointer points to inside a dynamically allocated memory.